<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProductForm</class>
 <widget class="QWidget" name="NewProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="sellingPriceLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sellingPriceLabel">
       <property name="text">
        <string>Selling Price</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="skuLabel">
       <property name="text">
        <string>SKU</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="purchaseLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="descriptionTextEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="skuLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="purchasePriceLabel">
       <property name="text">
        <string>Purchase Price</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="unitsLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="unitsLabel">
       <property name="text">
        <string>Units</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="TextEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
